intro-HPC/title.tex

```diff
@@ -5,7 +5,7 @@
 \vspace*{1.5\baselineskip}
 
 \Huge \strong{HPC Tutorial} \\
-\LARGE Version 20180906.03
+\LARGE Last updated: \today
```

Comment:
@redfast00 This isn't the right way to do it, since then all PDFs will get the same timestamp on deploy, it should be connected to the last made change per document...